test/src/test/java/hudson/model/AbstractItemSecurityTest.java

```diff
@@ -59,7 +61,7 @@ public void testUpdateByXmlDoesNotProcessForeignResources() throws Exception {
         FreeStyleProject project = jenkinsRule.createFreeStyleProject("security-167");
         project.setDescription("Wibble");
         try {
-            project.updateByXml(new StreamSource(new StringReader(xml)));
+            project.updateByXml((Source)new StreamSource(new StringReader(xml)));
```

Comment:
What would be the motivation behind this?

Reply:
`public void updateByXml(StreamSource source)` is deprecated, and the call within this deprecated method is `updateByXml((Source)source);`